build.gradle

```diff
@@ -10,18 +10,31 @@ archivesBaseName = project.archives_base_name
 version = project.mod_version
 group = project.maven_group
 
+repositories {
+    mavenCentral()
+    maven {
+        name = "impactdevelopment-repo"
+        url = "https://impactdevelopment.github.io/maven/"
```

Comment:
Why using the repository of another (third-party) mod?

Reply:
gimme a sec, I forget why rn

Reply:
ah, it's because something in the fabritone repo requires it for compiling. And for some reason it won't import it unless it's in our build.gradle